Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -7466,6 +7466,195 @@ script:
+  - arguments:
+    - default: false
+      defaultValue: null
+      description: String to filter to only check given device
+      isArray: false
+      name: device_filter_string
+      required: false
+      secret: false
+    deprecated: false
+    description: Checks at least one Log forwarding profile is configured according to best practices.
+    name: pan-os-hygiene-check-log-forwarding
+    outputs:
+    - contextPath: PANOS.ConfigurationHygiene.Summary.description
+      description: The description of the check
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Summary.issue_code
+      description: The shorthand code for this hygiene check
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Summary.result
+      description: Whether the check passed or failed
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Summary.issue_count
+      description: Total number of matching issues
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Result.hostid
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Result.container_name
+      description: What parent container (DG, Template, VSYS) this object belongs
+        to.
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Result.issue_code
+      description: The shorthand code for the issue
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Result.description
+      description: Human readable description of issue
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Result.name
+      description: The affected object name
+      type: String
+  - arguments:
+    - default: false
+      defaultValue: null
+      description: String to filter to only check given device
+      isArray: false
+      name: device_filter_string
+      required: false
+      secret: false
+    - default: false
+      defaultValue: critical,high
+      description: csv list of severities that must be in drop/reset/block-ip mode.
+      isArray: false
+      name: minimum_block_severities
+      required: false
+      secret: false
+    - default: false
+      defaultValue: medium,low
+      description: csv list of severities that must be in alert/default or higher
+        mode.
+      isArray: false
+      name: minimum_alert_severities
+      required: false
+      secret: false
+    deprecated: false
+    description: Checks the configured Vulnerability profiles to ensure at least one meets best practices.
+    name: pan-os-hygiene-check-vulnerability-profiles
+    outputs:
+    - contextPath: PANOS.ConfigurationHygiene.Summary.description
+      description: The description of the check
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Summary.issue_code
+      description: The shorthand code for this hygiene check
+      type: String
+    - contextPath: PANOS.ConfigurationHygiene.Summary.result
+      description: Whether the check passed or failed
+      type: String
```

Comment:
are you sure its not a boolean here?

Reply:
This one is actually a unicode string which renders into  a nice little ✔️  in the readable output on XSOAR :)

Reply:
nice